### PR TITLE
Load member .env file that is generated by etcdadm.

### PR DESCRIPTION
**Pull request by martinssipenko**

### CODE REVIEW

File: core/controlplane/config/templates/cloud-config-etcd [hunk 1]
```diff
@@ -269,6 +269,7 @@ coreos:
 
             [Service]
             EnvironmentFile=-/etc/etcd-environment
+            EnvironmentFile=-/var/run/coreos/etcdadm/etcd-member.env
```
Comment (mumoshu) [inline core/controlplane/config/templates/cloud-config-etcd]: @martinssipenko Hi, thanks for contributing!
Would you mind sharing me your intention to add this line?
Seemed like this env file is read from nowhere.

Comment (martinssipenko) [reply core/controlplane/config/templates/cloud-config-etcd]: This file is generated by etcdadm

Comment (mumoshu) [reply core/controlplane/config/templates/cloud-config-etcd]: Yes! However, I thought we already feed it to etcd somewhere else?
Where is the env file used from when written here?

**Comment by martinssipenko** [comment]
Its mever being fed to etcd, at least i could not find a place where it does. There was an old open pull request that roughly did the same, but it seems to be abandoned, i took few parts from the old PR.

Martins

**Comment by mumoshu** [comment]
@martinssipenko Thanks for clarifying! Yes, you are correct. I was missing the fact that the newly added EnvironmentFile statement was inside the drop-in for the etcd-member.service. So it might work, of course!

**Comment by mumoshu** [comment]
Thanks for contributing :bow:

**Pull request merged**